src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -78,6 +78,44 @@ end
 SimpleDiGraph(nv::T, ne::Integer; seed::Int=-1) where T <: Integer =
     SimpleDiGraph{Int}(nv, ne, seed=seed)
 
+"""
+	SimpleDiGraph(g, p_in, p_out; seed=-1)
+
+Create a directed graph from an undirected graph `g` where the direction of each
+edge from `u` to `v` (`u < v`) in `g` is added as a directed edge from `u` to `v`
+with probability `p_out`, from `v` to `u` with probability `p_in`, or in both
+directions with probability `1 - p_in - p_out`. If `p_in + p_out ≥ 1`, `p_out`
+will be truncated to `1 - p_in`, and no bidirectional edges will be created.
+"""
```

Comment:
It would be helpful, if you specified here, that the order is taken by viewing the vertices as integers, otherwise it is not that easy to comprehend I guess.

Reply:
What else could they be? `SimpleGraphs` requires integer vertices (it's in the documentation).